client/src/pages/Availability/Availability.tsx

```diff
@@ -0,0 +1,297 @@
+import { Button, InputLabel, FormControl } from '@mui/material';
+import { Box } from '@mui/system';
+import SettingHeader from '../../components/SettingsHeader/SettingsHeader';
+import { Typography } from '@mui/material';
+import Select from '@mui/material/Select';
+import MenuItem from '@mui/material/MenuItem';
+import CreateAvailInterface from './AvailInterface';
+import { Formik, FormikHelpers } from 'formik';
+import React, { useState, useEffect } from 'react';
+import FormInput from '../../components/FormInput/FormInput';
+import CreateSchedule from '../../helpers/APICalls/createSchedule';
+import { useAuth } from '../../context/useAuthContext';
+import { useSnackBar } from '../../context/useSnackbarContext';
+import GetSchedule from '../../helpers/APICalls/getSchedules';
+import getSchedulesById from '../../helpers/APICalls/getScheduleById';
+import { AvailabileValue } from '../../interface/AvailabilityApiData';
+import Record from './Records';
+const week = ['monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday', 'sunday'];
+
+export default function Availability(): JSX.Element {
+  const [schedules, setSchedules] = useState([{ name: '', id: 'Select' }]);
+  const [newSchedule, setNewSchedule] = useState(false);
+  const [daysInfo, setDaysInfo] = useState<any>([]);
+  const { updateLoginContext } = useAuth();
+  const { updateSnackBarMessage } = useSnackBar();
+  const [selectedSchedule, setSelectedSchedule] = useState('');
+  const [check, setCheck] = useState(1);
+  const [dropDownSelected, setDropDownSelected] = useState(false);
+  const [available, setAvailable] = useState<AvailabileValue>();
+  const generateSchedules = (scheduleName: any, id: any) => {
+    return (
+      <MenuItem
+        onClick={(e) => {
+          getSelectedScheduleById(e.currentTarget.id);
+          setDropDownSelected(true);
+        }}
+        key={id}
+        id={id}
+        value={scheduleName}
+      >
+        {scheduleName}
+      </MenuItem>
+    );
+  };
+  const handleChange = (e: any) => {
+    setSelectedSchedule(e.target.value);
+    setNewSchedule(false);
+  };
+  const fillDataInfo = async (data: AvailabileValue) => {
+    // available.push(data);
```

Comment:
Please uncomment or remove commented code

Reply:
Done the changes

Reply:
Done the changes